templates/complete/main.tf

```
@@ -69,23 +69,25 @@ module "hubnetworking" {
 }
 
 module "virtual_network_gateway" {
-  source  = "Azure/vnet-gateway/azurerm"
-  version = "0.1.2"
+  source  = "Azure/avm-ptn-vnetgateway/azurerm"
+  version = "0.2.0"
```

Should we take minor versions by default?

```suggestion
  version = "~> 0.2.0"
```